### DISCUSSION TITLE
無程式碼時代來臨

### DISCUSSION THREAD

**Discussion by s0973913299** [1101-GS3073-C/Python-Community-GS3073-C#26] [General]
未來撰寫程式碼的門檻會越來越低

在疫情的影響下 很多的服務或是都需要轉型數位化 但是IT人才沒那麼多 無程式碼開發可能就是趨勢 所以no code撰寫未來會變成一項基礎能力

[]()https://www.bnext.com.tw/article/65287/no-code-generation-is-coming

**Comment by chia-kai-chang** [comment]
@s0973913299 你可以舉出三個no-code machine learning platform的例子嗎?

**Reply by s0973913299** [reply]
Airtable
FileMaker
Webflow

**Reply by chia-kai-chang** [reply]
## 從Gartner的市場調查數據來看
根據的Gartner市調結果，這些是2021年在「Data Science and Machine Learning Platform的領導平台」
![image](https://user-images.githubusercontent.com/1768891/135983138-cb6612fa-b684-4c0a-a244-b2a9bf9132f4.png)


## 想要動手玩
如果大家想玩玩no-code machine learning的話，建議可以玩玩：
1. [Knime](https://www.knime.com/)：開源免費，可免費下載安裝
2. [dataiku.com](dataiku.com) ：網頁版
 
另外，H2o.ai跟DataRobot兩年前蠻有名的，雖然技術依然不錯，但是易用度變差了。

## 純粹分享閒聊
另外附上2018年參加ODSC商業展覽的照片：

![IMG_5435](https://user-images.githubusercontent.com/1768891/135989541-e7ee1e23-4c03-44f5-b82d-235b72443b49.jpg)


![IMG_5438](https://user-images.githubusercontent.com/1768891/135989554-a288ec23-bbf2-43fc-a9c6-6504fa42f164.jpg)


![IMG_5443](https://user-images.githubusercontent.com/1768891/135989568-25443e4b-9a67-49dd-b41a-5b4bd9524881.jpg)


![IMG_5524](https://user-images.githubusercontent.com/1768891/135989579-a2803bcf-70d5-475b-a58c-08b5bdb5e0e0.jpg)


![IMG_5527](https://user-images.githubusercontent.com/1768891/135989593-b890408b-e405-40ec-b22f-2f56b5fdb2e6.jpg)

**Comment by chia-kai-chang** [comment]
微軟也在no-code ML領域深耕多年。畢竟word, excel當年就是為了取代複雜的pe2?

現在微軟在推 **power flow**, **power BI**, **power apps**，大家有空可以搜尋一下這些平台，可以帶來甚麼樣的效率提升。另外熟悉power平台還可以考微軟證照喔。
